### PR TITLE
Fix: Bosyun.BoardMix version 1.3.8

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100180)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@
 PackageIdentifier: Bosyun.BoardMix
 PackageVersion: 1.3.8
 PackageLocale: en-US
-Publisher: BoardMix
+Publisher: boardmix
```
Comment (stephengillie) [inline manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml:7]:
```suggestion
Publisher: Bosyun
```

File: manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.zh-CN.yaml [hunk 2]
```diff
@@ -4,12 +4,12 @@
 PackageIdentifier: Bosyun.BoardMix
 PackageVersion: 1.3.8
 PackageLocale: zh-CN
-Publisher: BoardMix
+Publisher: boardmix
```
Comment (stephengillie) [inline manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.zh-CN.yaml:7]:
```suggestion
Publisher: Bosyun
```

File: manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml:7]: I got this:
image

Comment (SpecterShell) [reply manifests/b/Bosyun/BoardMix/1.3.8/Bosyun.BoardMix.locale.en-US.yaml:7]: @stephengillie

**Review by stephengillie — APPROVED** [review]

**Pull request merged**